src/receive_imf.rs

```diff
@@ -388,6 +388,9 @@ pub async fn from_field_to_contact_id(
     }
 }
 
+/// Creates a `ReceivedMsg` from given parts which might consist of
+/// mulitple messages, depending on the attachements.
+/// Every entry in `mime_parser.parts` produces a new row in the `msgs` table.
```

Comment:
```suggestion
/// Creates a `ReceivedMsg` from given parts which might consist of
/// mulitple messages (if there are multiple attachments).
/// Every entry in `mime_parser.parts` produces a new row in the `msgs` table.
```
or:
```suggestion
/// Adds a new row in the `msgs` table for every entry in `mime_parser.parts`.
/// There are mulitple messages i.e. multiple parts (if there are multiple attachments).
```

Since the `ReceivedMsg` is just the return type, including some information about the added message, but I wouldn't say it's the most important thing this function does (though I e.g. wasn't there when the name `add_parts` was chosen, and this can be discussed.)